presidio-databinder/cmd/presidio-databinder/database/database-binder.go

```diff
@@ -0,0 +1,90 @@
+package databaseBinder
+
+import (
+	"fmt"
+	"time"
+
+	// import mssql driver
+	_ "github.com/denisenkom/go-mssqldb"
+	// import mysql driver
+	_ "github.com/go-sql-driver/mysql"
+	"github.com/go-xorm/xorm"
+	// import postegrsql driver
+	_ "github.com/lib/pq"
+	// import sqlite driver
+	_ "github.com/mattn/go-sqlite3"
+
+	message_types "github.com/presid-io/presidio-genproto/golang"
+	log "github.com/presid-io/presidio/pkg/logger"
+	"github.com/presid-io/presidio/presidio-databinder/cmd/presidio-databinder/databinder"
+)
+
+type dBDataBinder struct {
+	driverName       string
+	connectionString string
+	engine           *xorm.Engine
+	tableName        string
+}
+
+// New returns new instance of DB Data writter
+func New(driverName string, connectionString string, tableName string) databinder.DataBinder {
+	// default table name
+	if tableName == "" {
+		tableName = "scannerresult"
```

Comment:
should be configurable

Reply:
it's configurable through the scanner template. this is the default in case that it wasn't supplied in the scanner template.